app/helpers/fhir_helper.rb

```diff
@@ -308,6 +312,84 @@ def vaccine_from_fhir(vaccine)
     }
   end
 
+  # Returns a representative FHIR::Observation for an instance of a Sara Alert Laboratory.
+  # https://www.hl7.org/fhir/observation.html
+  def laboratory_as_fhir(laboratory)
+    coded_lab_type = Laboratory.lab_type_to_code(laboratory.lab_type)
+    coded_result = Laboratory.result_to_code(laboratory.result)
+
+    FHIR::Observation.new(
+      meta: FHIR::Meta.new(lastUpdated: laboratory.updated_at.strftime('%FT%T%:z')),
+      id: laboratory.id,
+      status: 'final',
+      category: [
+        FHIR::CodeableConcept.new(
+          coding: [
+            FHIR::Coding.new(
+              system: 'http://terminology.hl7.org/CodeSystem/observation-category',
+              code: 'laboratory'
+            )
+          ]
+        )
+      ],
+      code: FHIR::CodeableConcept.new(
+        text: laboratory.lab_type,
+        coding: [
+          FHIR::Coding.new(
+            system: coded_lab_type[:system],
+            code: coded_lab_type[:code]
+          )
+        ]
+      ),
+      subject: FHIR::Reference.new(reference: "Patient/#{laboratory.patient_id}"),
+      effectiveDateTime: laboratory.specimen_collection&.strftime('%FT%T%:z'),
+      valueCodeableConcept: coded_result.nil? ? nil : FHIR::CodeableConcept.new(
+        text: laboratory.result,
+        coding: [
+          FHIR::Coding.new(
+            system: coded_result[:system],
+            code: coded_result[:code]
+          )
+        ]
+      ),
+      extension: [
+        to_date_extension(laboratory.report, 'report-date')
```

Comment:
I agree with your inference, I think `issued` works well here. I think in the context of our system, the providers are the public health officials, and so the report date is the date that the observation is made available.

Reply:
Actually one thing that is making me second guess things is that the type of `issued` is `instant`, where we only accept a `date`. Since our data type is so much less specific, it might be a little weird to use `issued`, since we are somewhat misrepresenting what sort of data we accept. What do you think?

Reply:
Good point, this is tough. After going back and forth on this, I think the pros of not having an extension outweigh the cons of mildly abusing `instant`. So I would just ignore the time component when taking in this value, and then return it with 00:00:00. Still, it would probably be good to document this in the IG.

Reply:
Done in 33b6eee.